fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/keyspace/ResolvedKeySpacePath.java

```diff
@@ -279,7 +278,6 @@ public static void appendValue(StringBuilder sb, Object value) {
      * @return a new {@code ResolvedKeySpacePath} that is the same as this, except with a different {@link #getRemainder()}.
      */
     @Nonnull
-    @VisibleForTesting
```

Comment:
if `getResolvedPath` is removed from `DataInKeyspacePath`, then maybe this method can be removed as well (or stay as `VisibleForTesting`?)